docs/openstack/create-and-connect-to-the-VM/ssh-to-the-VM.md

```diff
@@ -0,0 +1,320 @@
+[launching the instance](launch-a-VM.md).
+
+- [Assign a Floating IP](assign-a-floating-IP.md) to the instance in order to
+access it from outside world.
+
+- Make sure you have added rules in the
+[Security Groups](../access-and-security/security-groups.md#allowing-ssh) to
+allow **ssh** using Port 22 is opened to the instance.
+
+!!! info "How to update New Security Group(s) on any running VM?"
+    If you want to attach/deattach any new Security Group(s) to/from a running VM
+    after it has launched. First create all new Security Group(s) with all the rules
+    required. Following [this guide](../access-and-security/security-groups.md#update-security-groups-to-a-running-vm),
+    you'll be able to attach any newly created security group(s) with all the
+    required rules to a running VM.
+
+Make a note of the Floating IP you have associated to your instance.
+
+![Associated Instance Floating IP](images/floating_ip_is_associated.png)
+
+In our example, the IP is `199.94.60.66`.
+
+Default usernames for all the base images are:
+
+- **all Ubuntu images**: ubuntu
+- **all CentOS images**: centos
+- **all Rocky Linux images**: rocky
+- **all Fedora images**: fedora
+- **all Debian images**: debian
+- **all RHEL images**: cloud-user
+
+Our example VM was launched with the **ubuntu-22.04-x86_64** base image, the
+user we need is 'ubuntu'.
+
+Open a Terminal window and type:
+
+    ssh ubuntu@199.94.60.66
+
+Since you have never connected to this VM before, you will be asked if you are
+sure you want to connect. Type `yes`.
+
+![SSH To VM Successful](images/ssh_to_vm.png)
+
+!!! tip "Note"
+    If you haven't added your key to ssh-agent, you may need to specify the
+    private key file, like this: `ssh -i ~/.ssh/cloud.key ubuntu@199.94.60.66`
+
+---
+
+## Setting a password
+
+When the VMs are launched, a strong, randomly-generated password is created for
+the default user, and then discarded.
+
+Once you connect to your VM, you will want to set a password in case you ever
+need to log in via the console in the web dashboard.
+
+For example, if your network connections aren't working right.
+
+!!! info "Setting a password is necessary to use Remote Desktop Protocol (RDP)"
+    [Remote Desktop Protocol](https://learn.microsoft.com/en-us/troubleshoot/windows-server/remote/understanding-remote-desktop-protocol)
+    (RDP) is widely used for Windows remote connections, but you can also access
+    and interact with the graphical user interface of a remote Linux server by
+    using a tool like [xrdp](http://xrdp.org/), an open-source implementation of
+    the RDP server. You can use `xrdp` to remotely access the Linux desktop. To
+    do so, you need to utilize the RDP client. Moreover, xrdp delivers a login
+    to the remote machines employing Microsoft RDP. This is why a user with
+    password is necessary to setup on such VM. You can refer to [this guide](#how-to-enable-remote-desktop-protocol-using-xrdp-on-ubuntu)
```

Comment:
password is necessary to setup on such VM. -> the password is necessary to access the VM.